Pear/Controller/FeedbackViewController.swift

```diff
@@ -310,3 +387,11 @@ extension FeedbackViewController: UICollectionViewDelegateFlowLayout {
     }
 
 }
+
+private class FinishButton: UIButton {
+    override public var isEnabled: Bool {
+        didSet {
+            backgroundColor = isEnabled ? .backgroundOrange : .inactiveGreen
```

Comment:
I think there might be some shadow added too when the button is enabled.

Reply:
Good catch! I missed that in the Figma